eZ/Publish/API/Repository/Values/Content/TrashItem.php

```diff
@@ -19,4 +19,15 @@ abstract class TrashItem extends Location
      * @var \DateTime
      */
     protected $trashed;
+
+    /** @var array<int, int> */
+    protected $removedLocationContentIdMap = [];
+
+    /**
+     * @return array<int, int>
+     */
+    public function getRemovedLocationContentIdMap(): array
```

Comment:
In https://github.com/ezsystems/date-based-publisher/pull/267 I see that you are using this map to create content deletion notifications, and perform actual deletions.

Note that Content can have multiple locations, which means the same Content ID might occur in this map multiple times. Is this your intent?

Reply:
@Steveb-p Yes, it can appear multiple times. Why would that be a problem? If one schedule entry is deleted, the second won't be found and hence, won't be deleted. No exception is being thrown.